### ISSUE TITLE
About hyper‘s multi-tenant

### ISSUE THREAD

**Issue by keontang** [hyperhq/hypernetes-docs#6]
How does hyper achieve multi-tenant？ One tenant one hypernetes cluster？

**Comment by feiskyer** [comment]
Nope, all tenants are sharing same hypernetes cluster. hypernetes manages tenant via openstack keystone and creates a L2-isolated network for each tenant.

**Comment by keontang** [comment]
OK, now i see that one tenant just has several k8s namespace.
And for service, shall we set lbaas's ip according to service's vip?

**Comment by feiskyer** [comment]
Hypernetes will set service's external IP according to lbaas's floating ip.

**Comment by keontang** [comment]
How about cluster ip？It will be set according to lbaas's vip?

**Comment by feiskyer** [comment]
cluster ip is load balanced via haproxy (or ipvs for latest version). it's not in the same subnet of lbaas's vip. Please visit https://docs.hypernetes.com/user-guide/user-guide.html for usage.

**Comment by keontang** [comment]
I wander to know if each tenant will have a haproxy for cluster ip.

**Comment by feiskyer** [comment]
there is a haproxy for each pod before hypernetes v1.3, but we have switched it to ipvs (inside each VM) in latest version.

**Comment by keontang** [comment]
I am more confused, hypernetes just used openstack keystone, neutron and cinder but not nova, where did vm come from? Does hypercontainer run on vm？
